### ISSUE TITLE
What should the default settings for sound be?

### ISSUE THREAD

**Issue by jbphet** [phetsims/tambo#17]
As of this writing we have two global parameters for controlling sound, one that controls whether it is globally enabled or disabled, and one that controls whether the "basic" or "enhanced" level of sonification is provided.  We should decide what the default values are for these parameters.

I think we've essentially agreed that the default value for the level should be "basic", so please speak up if you disagree with that, otherwise I'll go with that.

I'm less clear on how enabled/disabled should work by default.  In our currently published sims that provide sound for games, the default is to have sound generation enabled.  In at least one sim - Plinko Probability - sound generation is off by default.  My thought is that we would probably have sound enabled by default, but the bouncing of the balls would be part of the "enhanced" level in sims like Plinko so that they don't produce too much sound in the default configuration.

@emily-phet and @ariel-phet - can I get your input on this?

**Comment by ariel-phet** [comment]
@jbphet I agree that the default value between "enhanced" and "basic" should be "basic" - that seems to fit with many of the discussions we have had.

As for sound on or off by default, that seems a bit trickier. I feel like PhET has gone back and forth on this question on different sims. I think for instance games often have sound on by default, whereas some sims like Pinko have it off. I don't have a strong opinion here, especially if we have a clear universal sound control. It seems sound on by default might be the easiest, and just let people turn it off or mute if the don't want it. But we could also do a case by case basis as we have been.

**Comment by jbphet** [comment]
The consensus at this point is to have sound on by default, set to "basic", and the user has to use the PhET menu to turn on "enhanced" sounds.  We have decided that there should be a query parameter that can prevent sound from being enabled if it is turned on in the sim flags, but that will be handled in a different issue.  These initial settings have been implemented, so I think it's safe to close this.
